client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -69,6 +84,15 @@ function EmailPlanMailboxesList( { emails, isLoadingEmails } ) {
 						} ) }
 					</Badge>
 				) }
+				{ isEmailForward( email ) && (
+					<>
+						<Gridicon icon="chevron-right" />
+						<span> { getEmailForwardAddress( email ) } </span>
+						{ ! isEmailForwardVerified( email ) && (
+							<MailboxListItemWarning warningText={ translate( 'Verification required' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Verification required for domains' )` *ES Score: 7*